src/thorlabs_cube/driver/kcube/kpa.py

```diff
@@ -0,0 +1,400 @@
+import struct as st
+
+from thorlabs_cube.driver.base import _Cube
+from thorlabs_cube.driver.message import MGMSG, Message, MsgError
+
+
+_CHANNEL: int = 0x01
+
+
+class Kpa(_Cube):
+    """KPA101 driver implementation."""
+
+    def __init__(self, loop, serial_dev) -> None:
+        """Initialize the KPA101 driver.
+
+        :param loop: Event loop for asynchronous operations.
+        :param serial_dev: Serial device identifier.
+        """
+        _Cube.__init__(self, loop, serial_dev)
+        self.loop_params = None
+
+    async def handle_message(self, msg: Message) -> None:
+        """Handle incoming messages from the KPA101 device.
+
+        :param msg: Message object received from the device.
+        """
+        msg_id = msg.id
+        data = msg.data
+
+        if msg_id == MGMSG.HW_DISCONNECT:
+            raise MsgError("Error: Please disconnect the KPA101")
+        
+        elif msg_id == MGMSG.HW_RESPONSE:
+            raise MsgError("Hardware error, please disconnect and reconnect the KPA101")
+        
+        else:
+            raise MsgError(f"Unhandled message ID: {msg_id}")
```

Comment:
Does this method need to handle the `MGMSG.HW_RICHRESPONSE` case (like e.g. the [KDC101](https://github.com/quantumion/thorlabs_cube/blob/f2d1f22d96a45579f36d088bb3ab248f04223caa/src/thorlabs_cube/driver/kcube/kdc.py#L30))?